_data/versions.yml

```diff
@@ -1346,6 +1346,68 @@ v2.1:
       version: 1.4.1
       url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=1.4.1
 
+v3.0:
+- title: v3.0.0-beta-rc1
+  note: |
+    20 October 2017
+
+    Calico v3.0.0-beta is a beta release that adds fixes and enhancements
+    to the Kubernetes integration. Enhancements include:
+    - Support for etcdv3 datastore
+    - A new API tuned for etcdv3 - see [libcalico-go release notes](https://github.com/projectcalico/libcalico-go/releases) for more details.
+
+    <div class="alert alert-danger" role="alert"><b>Important</b>: v3.0.0-beta has the following limitations:</div>
+    - **Only Kubernetes integration is currently supported.** `OpenStack`, `Docker`, and `Mesos`
+      are not supported in v3.0 - bad things will happen if you upgrade! This limitation
+      will also be true for the v3.0 GA release. Support for integrations other than Kubernetes
+      will follow soon after v3.0.
+    - Currently only **new** Kubernetes installations are supported. For existing deployments,
+      migrating from `etcdv2` to `etcdv3` is required, but not currently supported.
+    - There is limited data validation, therefore it may be easy to misconfigure resources or policy.
+    - Resource definitions have changed - old `calicoctl` manifests will need to be updated. See
+      [Calicoctl resource definitions](http://0.0.0.0:4000/v3.0/reference/calicoctl/resources/) for details.
```

Comment:
"http://0.0.0.0:4000" --> "https://docs.projectcalico.org"

Reply:
good catch

Reply:
Or.....you could use a relative link (recommended)

Reply:
TBH, I think that this would be the place to explain just what has changed....not the reference section.